Neos.Flow/Tests/Behavior/Features/Bootstrap/SecurityOperationsTrait.php

```diff
@@ -19,6 +18,7 @@
 use Neos\Utility\Arrays;
 use PHPUnit\Framework\Assert;
 use Psr\Http\Message\ServerRequestFactoryInterface;
+use Symfony\Component\Yaml\Yaml;
```

Comment:
Where is this new import used?

Reply:
Ah, I'll remove this. This was introduced as we tried to avoid generating a temporary test policy file as it is currently the case (https://github.com/neos/flow-development-collection/commit/9a0e9d8a7d278a2790775ec48e370c6bc3184c4d#diff-43787f11e7a6f604003f41baab1eada8d5ee8399ac202e1f07157f5a9846da52).

Reply:
The `use ..Yaml` is removed.